activesupport/lib/active_support/inflector/inflections.rb

```diff
@@ -1,4 +1,6 @@
-require 'concurrent'
+# FIXME: clean up warning spam from concurrent lib
```

Comment:
I plan to publish a new build of concurrent-ruby today. That should fix the warnings. I'll create a Rails PR pointing to the new version as soon as I do.